src/main/java/io/jenkins/plugins/orka/client/ConfigurationRequest.java

```diff
@@ -51,8 +51,12 @@ public ConfigurationRequest(String vmName, String image, String baseImage, Strin
         this.configTemplate = configTemplate;
         this.cpuCount = cpuCount;
         this.scheduler = StringUtils.isNotBlank(scheduler) ? scheduler : null;
-        if (!StringUtils.isBlank(memory) && !StringUtils.equals(memory, "auto") && Integer.parseInt(memory) > 0) {
-            this.memory = Integer.parseInt(memory);
+        try {
+            if (!StringUtils.isBlank(memory) && !StringUtils.equals(memory, "auto") && Float.parseFloat(memory) > 0) {
```

Comment:
Isn't it better to throw an error here?
Instead of silently swallowing it?

Having an invalid memory here is not expected, so I wound throw an error.

Reply:
I see this as resolved, but do not see any changes :)